tests/raw_query/tests.py

```diff
@@ -129,6 +131,15 @@ def test_db_column_handler(self):
         coffees = Coffee.objects.all()
         self.assertSuccessfulRawQuery(Coffee, query, coffees)
 
+    def test_pk_with_mixed_case_db_column(self):
+        """
+        Test of a simple raw query against a model containing a pk with mixed
```

Comment:
I'd omit "Test of a" in the spirit of https://code.djangoproject.com/ticket/27392. "A raw query with a model that has a pk db_column with mixed case." seems good.